src/api/libcellml/component.h

```diff
@@ -16,8 +16,9 @@ limitations under the License.
 
 #pragma once
 
-#include "libcellml/componententity.h"
 #include "libcellml/exportdefinitions.h"
+#include "libcellml/componententity.h"
```

Comment:
I believe we did, at some point, agree on listing include files in alphabetical order. So, we should have `componententity.h` back to where it used to be.